languages/go/config.json

```diff
@@ -20,7 +20,6 @@
         "slug": "basics",
         "uuid": "6396fe40-ff11-4ff5-b880-9681bce30a6e",
         "concepts": ["basics"],
-        "prerequisites": ["comments"]
```

Comment:
`basics` is meant to be the very first exercise. It's explanation introduces packages and folders, etc. Basically all the basics needed to start with Go.

Reply:
got it!